pkg/doc/util/jwkkid/kid_creator.go

```diff
@@ -69,11 +80,13 @@ func buildJWK(keyBytes []byte, kt kms.KeyType) (*jose.JWK, error) {
 		if err != nil {
 			return nil, fmt.Errorf("buildJWK: failed to build JWK from ecdsa DER key: %w", err)
 		}
-	case kms.ED25519Type:
-		jwk, err = jose.JWKFromPublicKey(ed25519.PublicKey(keyBytes))
-		if err != nil {
-			return nil, fmt.Errorf("buildJWK: failed to build JWK from ed25519 key: %w", err)
-		}
+		// TODO remove `case kms.ED25519Type` in CreateKID() and uncomment below case when go-jose fixes Ed25519
```

Comment:
What is go Jose issue number

Reply:
there's no issue in go-jose repo, only a pending pr mentioned in the AFG issue #2530